chemprop/args.py

```diff
@@ -297,6 +297,8 @@ class TrainArgs(CommonArgs):
     """Aggregation scheme for atomic vectors into molecular vectors"""
     aggregation_norm: int = 100
     """For norm aggregation, number by which to divide summed up atomic features"""
+    #ensemble_variance: bool = False
+    #"""Whether to calculate the variance of ensembles as a measure of epistemic uncertainty. If True, the variance is saved as an additional column for each target in the preds_path."""
```

Comment:
You should remove these lines rather than commenting them out. I imagine you did this for testing and then didn't come back to them.

Reply:
Sorry about that. Yes that was just for testing. I'll fix that now.